worker/src/api/index.ts

```diff
@@ -116,7 +118,50 @@ router
     }
   });
 
-router.use("/emojis", emojis);
+router
+  .use(
+    basicAuth({
+      users: { admin: env.LANGFUSE_WORKER_PASSWORD },
+    })
+  )
+  .post("/ingestion", async (req, res) => {
+    try {
+      if (!redis) {
```

Comment:
Id wrap this in the retry function from the current ingestion pipeline to become more fault tolerant